import-scripts/pipelines_eks/mycrontab

```diff
@@ -6,6 +6,8 @@ MAILTO=cbioportal-pipelines@cbioportal.org
 * * * * * . /data/portal-cron/scripts/automation-environment.sh;/data/portal-cron/scripts/import-portal-users.sh || echo "Failure in crontab: import-portal-users.sh exited with non-zero exit status" | mail -r "cbioportal-pipelines@cbioportal.org" -s "Failure in crontab" cbioportal-pipelines@cbioportal.org
 * * * * * . /data/portal-cron/scripts/automation-environment.sh;/data/portal-cron/scripts/import-cmo-data-triage.sh >> /data/portal-cron/logs/import-cmo-data-triage.log 2>&1
 * * * * * . /data/portal-cron/scripts/automation-environment.sh;/data/portal-cron/scripts/kill-import-cmo-data-triage.sh >> /data/portal-cron/logs/kill-import-cmo-data-triage.log 2>&1
+* * * * * . /data/portal-cron/scripts/automation-environment.sh;/data/portal-cron/scripts/import-hgnc-data.sh >> /data/portal-cron/logs/import-hgnc-data.log 2>&1
```

Comment:
consolidate when we introduce GENIE?

Reply:
agreed .. we can consolidate this into a single script / crontab entry I believe